pulsar-functions/api-java/src/main/java/org/apache/pulsar/functions/api/Context.java

```diff
@@ -55,7 +55,7 @@
      * Get output Serde class
      * @return output serde class
      */
-    String getOutputSerdeClassName();
+    String getOutputSchemaType();
```

Comment:
Currently its confusing whether we call things serde or schema. We might want to clear this confusion

Reply:
I think we should just mention schema and leave serde for compatibility only.